ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Node/Serialisation.hs

```diff
@@ -85,6 +91,10 @@ class SerialiseNodeToClient blk a where
     => CodecConfig blk -> BlockNodeToClientVersion blk -> forall s. Decoder s a
   decodeNodeToClient _ccfg _version = decode
 
+deriving newtype instance
+     SerialiseNodeToClient blk ( qq     (query blk))
+  => SerialiseNodeToClient blk ((qq :.: query) blk)
```

Comment:
What is qq?

Reply:
Anything that composes with `query` :stuck_out_tongue:

Reply:
Removed